lib/js/jsconsole.nim

```diff
@@ -0,0 +1,32 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2012 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## Wrapper for the `console` object for the `JavaScript backend
+## <backends.html#the-javascript-target>`_.
+
+when not defined(js) and not defined(Nimdoc):
+  {.error: "This module only works on the JavaScript platform".}
+
+type Console* {.importc.} = ref object of RootObj
+
+{.push importcpp .}
+
+proc log*[A](console: Console, a: A)
```

Comment:
Actually this should be pretty straightforward, doing a custom overloaded conversion proc from anything to some `RootRef`, but now that I think of if. Does it make a lot of sense to dump nim objects right through js console without converting them to a more meaningful string? Nim objects don't look that pretty in js as you may know.
As a side note, are you aware of `console.SystemLogger` that already supports js console and you just use nim's existing `logging` facilities and get the job done? The only thing that `console.SystemLogger` always uses `console.log` regardless log level, but you might want to contribute to that to support `error`, `warn`, etc.